### DISCUSSION TITLE
New modes progress

### DISCUSSION THREAD

**Discussion by fudoido** [Courseplay/Courseplay_FS22#1843] [Q&A] [answered]
Hello team. Curious to know how is the development of the previous modes of CP for fs22.
Fieldwork operations seems to be quite stable currently, so i am thinking that maybe there is some start on the other modes, such as silo works and shovel works maybe? Is it something you guys still plan to add into fs22 CP? or the new game makes it harder to implement and it will be scrapped from fs22 CP?
Thanks in advance for any pointers about this :)

**Comment by Tensuko** [comment, marked as answer]
We wont give any progress information and dates.
